src/palindrom-event-target.js

```diff
@@ -0,0 +1,33 @@
+/*! Palindrom
+ * https://github.com/Palindrom/Palindrom
+ * (c) 2019 Omar Alsahker
+ * MIT license
+ */
+
+// privatize the prop
+const listeners = Symbol('listeners');
+
+export class PalindromEventTarget {
+    constructor() {
+        this[listeners] = {};
+    }
+    addEventListener(name, callback) {
+        if (!this[listeners][name]) {
+            this[listeners][name] = new Set();
+        }
+        this[listeners][name].add(callback);
+    }
+    removeListener(name, callback) {
```

Comment:
If class is supposed to mimic the EventDispatcher API, this method should be called `removeEventListener`.